packages/playwright/tests/options.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test } from '../src';
```

Comment:
Ohhh do these pages do something that the chromatic options are needed to fix? Very nice!

Reply:
@tevanoff thanks! Yeah, for the most part they "fail" when the option is not passed (visually at least) and in the snapshotted UI is a description of what "should" be seen.